internals/webpack/webpack.base.babel.js

```diff
@@ -4,6 +4,30 @@
 
 const path = require('path');
 const webpack = require('webpack');
+const dotenv = require('dotenv').config(); // eslint-disable-line no-unused-vars
```

Comment:
I would prefer the first option, while requiring at the top of the file (like ```import 'babel-polyfill') ```). But that's just my personal preference) And I'm not a maintainer)